backend/api/Areas/Leases/Controllers/SecurityDepositController.cs

```diff
@@ -117,6 +118,22 @@ public IActionResult DeleteDeposit(long leaseId, [FromBody] ParentConcurrencyGua
 
             return new JsonResult(_mapper.Map<LeaseModel>(updatedLease));
         }
+
+        /// <summary>
+        /// update the deposit note on the given lease
+        /// </summary>
+        /// <returns></returns>
+        [HttpPut("{leaseId:long}/deposits/note")]
```

Comment:
I am guessing this should be for only one desposit ? {leaseId:long}/deposits/{depositId}/note

Reply:
unfortunately, backmerges aren't really the time to fix these sorts of things. Can you log tech debt for this?